app/src/main/java/de/mathfactory/mooltifill/MooltifillService.kt

```diff
@@ -182,14 +200,32 @@ class MooltifillService : AutofillService() {
 
     private fun getHint(node: AssistStructure.ViewNode): String? {
         // return first autofill hint, if present
-        node.autofillHints?.firstOrNull()?.let { return it }
+        node.autofillHints?.firstOrNull()?.let {
```

Comment:
Good find! The existing code didn't call inferHint() here. Still it only evaluates the first hint, we should change this line to search all node.autofillHints for an inferrable one, not only the first. If you want you can do this change, otherwise I'll do that tomorrow in a separate PR.

Reply:
Hmm. Okay. I'll see what I can do!

Reply:
No worries, I already created a PR: #28